lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -100,6 +110,10 @@ def expected_path(constant)
 
           File.join(
             constants.map do |name|
+              k, v = custom_transform_patterns.each.find do |k, _|
```

Comment:
Would it make sense to give proper names like ‘pattern’ and ‘transformed’?

Reply:
Indeed it is, Thank you. Change to ‘pattern’ and ‘transformed’